wix-embedded-mysql/src/main/java/com/wix/mysql/EmbeddedMysql.java

```diff
@@ -0,0 +1,116 @@
+package com.wix.mysql;
+
+import com.google.common.collect.Lists;
+import com.wix.mysql.config.MysqldConfig;
+import com.wix.mysql.config.RuntimeConfigBuilder;
+import com.wix.mysql.config.SchemaConfig;
+import com.wix.mysql.distribution.Version;
+import de.flapdoodle.embed.process.config.IRuntimeConfig;
+import de.flapdoodle.embed.process.distribution.Distribution;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import static java.lang.String.format;
+
+/**
+ * @author viliusl
+ * @since 07/06/15
+ */
+public class EmbeddedMysql {
+    protected final MysqldConfig config;
+    protected final MysqldExecutable executable;
+    protected final MysqldProcess process;
+    private AtomicBoolean isRunning = new AtomicBoolean(true);
+
```

Comment:
I like it - it's a clear pattern - builders are within classes they are building. Maybe it adds some code, but otherwise code is quite stupid, so it's manageable. EmbeddedMysql, MysqldConfig, SchemaConfig, etc.